.github/workflows/swift-toolchain.yml

```diff
@@ -304,16 +308,17 @@ jobs:
           path: ${{ github.workspace }}/BuildRoot/Library/sqlite-${{ inputs.swift_toolchain_sqlite_version }}/usr
 
   cmark_gfm:
-    # TODO: Build this on macOS or make an equivalent Mac-only job
-    if: inputs.build_os == 'Windows'
     runs-on: ${{ inputs.default_build_runner }}
 
     strategy:
       fail-fast: false
-      matrix:
-        arch: ['amd64', 'arm64']
+      matrix: ${{ fromJSON(inputs.host_matrix) }}
 
-    name: Windows ${{ matrix.arch }} CMark GFM
+    name: ${{ matrix.os }} ${{ matrix.arch }} CMark GFM
+
+    defaults:
+      run:
+        shell: pwsh
```

Comment:
Can we do defaults on the workflow rather than the jobs?

Reply:
Done.